schema/schema.go

```diff
@@ -323,7 +323,7 @@ func (schema *Schema) GetPluralURLWithParents() string {
 
 // GetDbTableName returns a name of DB table used for storing schema instances
 func (schema *Schema) GetDbTableName() string {
```

Comment:
I think some "legacy_table_names" option in config db section should do. What do you think?